Sources/Afterpay/Resources/IntroText.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  IntroText.swift
+//  Afterpay
+//
+//  Created by Scott Antonac on 25/8/21.
+//  Copyright © 2021 Afterpay. All rights reserved.
+//
+
+import Foundation
+
+public enum AfterpayIntroText: String {
+  case NONE = ""
```

Comment:
I think there's no reason for this to be SO SHOUTY. You can just use `none`, or if you're trying to avoid confusion with `Optional.none` you can use `empty`?

Reply:
It was more the fact that it's the only option that doesn't actually align with its value... but I agree - the style isn't great. Will update to `empty`.

Reply:
Addressed in 0a4e72a